libs/facilities/frontend/view/src/lib/components/detail/chart/chart.component.ts

```diff
@@ -0,0 +1,62 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input, OnInit  } from '@angular/core';
+import { themeSwitcher } from '@siemens/ix';
+import { convertThemeName, registerTheme } from '@siemens/ix-echarts';
+import { EChartsOption } from 'echarts';
+import * as echarts from 'echarts';
+import { NgxEchartsDirective } from 'ngx-echarts';
+
+@Component({
+	selector: 'lib-chart',
+	standalone: true,
+    imports: [ CommonModule, NgxEchartsDirective ],
+	templateUrl: './chart.component.html',
+	styleUrl: './chart.component.scss',
+})
+export class ChartComponent implements OnInit {
+
+    theme = convertThemeName(themeSwitcher.getCurrentTheme());
+
+    // this is how the values are supposed to look but these values are never used
+    @Input({required: true}) data = [ [ '2024-04-22T22:06:03.695Z', 69 ] ]
```

Comment:
When you set sth as required you should never ever give it a default value. This makes no sense. Pass the value over the parent component to feed it with the data.

Btw pls use input signals
https://v17.angular.io/guide/signal-inputs

That makes the app reactive